source/_integrations/nest.markdown

```diff
@@ -299,6 +299,46 @@ All Google Nest Cam models and the Google Nest Hello Video Doorbell support [Dev
 
 See [Automating Home Assistant](/getting-started/automation/) for the getting started guide on automations or the [Automation](/docs/automation/) documentation for full details.
 
+### Example
+
+The events for emitted to Home Assistant look like this..
```

Comment:
I would lean towards just including the automation details, since those are more user friendly.  I feel like folks familiar with events may be able to figure this out.

Reply:
Yeah, that makes sense. I've removed that section